src/frontends/tensorflow/CMakeLists.txt

```diff
@@ -4,6 +4,23 @@
 
 add_subdirectory(src)
 
+execute_process(
+    COMMAND ${Python3_EXECUTABLE} 
+                   ${CMAKE_CURRENT_SOURCE_DIR}/docs/check_supported_ops.py
+                   ${CMAKE_CURRENT_SOURCE_DIR}/src/op_table.cpp
+                   ${CMAKE_CURRENT_SOURCE_DIR}/docs/supported_ops.md
+    RESULT_VARIABLE SCRIPT_RESULT
+    OUTPUT_VARIABLE SCRIPT_OUTPUT
+    ERROR_VARIABLE SCRIPT_ERROR
+)
+
+# Check the result and handle exceptions
+if(SCRIPT_RESULT EQUAL 0)
+    message(STATUS "Python script executed successfully")
```

Comment:
looks like it's useless information for end users

Reply:
agree.
@ilya-lavrenov, do you know how to turn on this check only in CI and exclude this from local build?

Reply:
why should we separate into two cases CI and local runs?

You can check whether it's CI or not by env vars https://github.com/openvinotoolkit/telemetry/pull/50/files#diff-a6d0545ab64fd261f20823a9c1fcd2bef6b4b06e9d819834a942ffb4731810b1R279-R293

Reply:
That is because I don't want to crash build for TF FE developers who experiment with TF FE, for example.
Once he completes some feature then he needs to productize and update the documentation.

Reply:
Done